### DISCUSSION TITLE
First transcription always comes late

### DISCUSSION THREAD

**Discussion by LEAGUEDORA** [deepgram/community#852] [General help]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

So, we are an AI company that does AI outbound phone calls. We are using deepgram as one of our ASR. But, we are observing that always our first `is_final = true` comes always late like after 3-4 seconds. But from the 2nd time the transcriptions are faster

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

wss://api.deepgram.com/v1/listen?encoding=mulaw&sample_rate=8000&channels=1&interim_results=false&model=nova-2&language=en-IN

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

a35c99bb-beec-4b9b-b2fc-97fe7940dd72

### If possible, please attach an example audio file to reproduce the issue.

It is streaming so, I could not provide one

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
